src/Pusher.php

```diff
@@ -424,7 +300,7 @@ public static function build_auth_query_string(
 
         $auth_query_string = self::array_implode('=', '&', $params);
```

Comment:
We shouldn't need this now that we are returning `$params`.
If that's the case I'd also change the function name.